concent_api/core/models.py

```diff
@@ -71,6 +96,8 @@ class Subtask(Model):
     """
     Represents subtask states.
     """
+    objects = Manager()
```

Comment:
This is by default, please remove.

Reply:
This is not by default and without that Django can't use default manager

Reply:
I will check when tests will be working.

Reply:
@MichalW-CP It's a good idea to refer to docs in cases like this. Saves the reviewer some unnecessary checking. I remember that we went through these docs before:

https://docs.djangoproject.com/en/1.11/topics/db/managers/#default-managers
> If you use custom Manager objects, take note that the first Manager Django encounters (in the order in which they’re defined in the model) has a special status. Django interprets the first Manager defined in a class as the “default” Manager, and several parts of Django (including dumpdata) will use that Manager exclusively for that model. As a result, it’s a good idea to be careful in your choice of default manager in order to avoid a situation where overriding get_queryset() results in an inability to retrieve objects you’d like to work with.

Reply:
This makes the query much more complex and we don't need these columns 99% of the time.

Also, while extra columns added with `annotate()` are nice, they can get in the way if you're doing complex joins. Especially if you're joining again with the table that was already used in column definition.

Reply:
> This makes the query much more complex and we don't need these columns 99% of the time.

`get_queryset` could be then changed to be a custom method.

Reply:
I meant the underlying raw SQL query. And not just looking more complex. Actually having more joins and conditions.

Reply:
The raw SQL query would be the same, unless you use custom method.

Reply:
How could it be the same if the extra columns refer for example to `task_to_compute__timestamp`? It must result either in a join or in an extra query.